From: Carsten Schlote <schlote@vahanus.net>
Date: Tue, 11 Mar 2008 00:32:20 +0100
Subject: [PATCH] Fixed bitops.h for Coldfire V4E

Commented out assembly inline code for mc68020 or higher.
These mnemonics aren't available on Coldfire. Using C routines
now.

Signed-off-by: Carsten Schlote <schlote@vahanus.net>
---
 lib/ext2fs/bitops.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: e2fsprogs-1.42.4/lib/ext2fs/bitops.h
===================================================================
--- e2fsprogs-1.42.4.orig/lib/ext2fs/bitops.h
+++ e2fsprogs-1.42.4/lib/ext2fs/bitops.h
@@ -347,7 +347,7 @@ _INLINE_ __u16 ext2fs_swab16(__u16 val)
 #endif	/* i386 */
 
 #if ((defined __GNUC__) && !defined(_EXT2_USE_C_VERSIONS_) && \
-     (defined(__mc68000__)))
+     (defined(__mc68000__)) && !defined(__mcoldfire__))
 
 #define _EXT2_HAVE_ASM_BITOPS_
 
